### ISSUE TITLE
Help with animating a responsive backgroundimage

### ISSUE THREAD

**Issue by maxrho** [Prinzhorn/skrollr#345]
Hi!

First off - thanks @Prinzhorn, awesome lib.

Secondly, this is what i want to do 

``` html
<div class="wrapper" style="position: relative;">
  <div style="width: 100%; height 500px; z-index: 99; margin-bottom: 400px; position: relative;">
    some content
  </div>

  <div id="FOO" style="top: 500px; height: auto; width: 100%; position: absolute;  z-index: 50;>
    <img data-anchor-target="#FOO" data-0-bottom-top="top:0%" data-0-top-bottom="top:-100%">
  </div>
</div>

<div class="wrapper" style="position: relative;">
  <div style="width: 100%; height 500px; z-index: 99; margin-bottom: 400px; position: relative;">
    some content
  </div>
</div>
```

What I want: When my viewport reaches the top of #FOO, the image in #FOO should start with top: 0px; and be just below the first DIV. Then as I scroll down, when #FOO's bottom is at the top of my browserwindow, I want the IMG to be at bottom: 0px;

I can't seem to figure this out, tried using offsets, weird hacks and a bunch of ugly solutions. I could maybe hook onto the JS and do it ultra ugly, but how would you advice me to go about this problem using Skrollr? Can it be done?

TL;DR - How do i make a width: 100% height: auto Image move and show the whole image in a fixed height sneak peak window?

God this explanation sucks, but it's the best I can do.

BR,
Max

Edit: I was thinking it was more of a feature request, because I don't think it's doable, but I could be wrong.
